### DISCUSSION TITLE
v0.9.5

### DISCUSSION THREAD

**Discussion by aawsome** [rustic-rs/rustic#1376] [Announcements]
Hello `rustic`eans,

we are happy to release rustic 0.9.5. This release brings some improvements for accessing data in the repository: 
- `webdav` has been fixed and now also works with remote storages
- `mount` has been added for the linux version
- `dump` now supports to write `.tgz` and `.zip` and direct writes to a file (self-detecting the output format).

Other major improvements are improved error messages and the ability to directly filter snapshots using `jq` syntax.

Thanks a lot to everyone who contributed with code, issues or discussions!

# Changelog

### Features

- *(commands)* More dump options ([#1339](https://github.com/rustic-rs/rustic/pull/1339))
- shut down gracefully with ctrl+c ([#1364](https://github.com/rustic-rs/rustic/pull/1364))
- Add --filter-jq option ([#1372](https://github.com/rustic-rs/rustic/pull/1372))
- *(commands)* Add `mount` command ([#973](https://github.com/rustic-rs/rustic/pull/973))
- Error messages are now much improved ([rustic_core](https://github.com/rustic-rs/rustic_core/releases/tag/rustic_core-v0.6.0))
- *(commands)* Add minutely timeline ([rustic_core](https://github.com/rustic-rs/rustic_core/releases/tag/rustic_core-v0.7.2)) to forget command

### Bugfixes

- *(commands)* run backup hooks before checking source dir ([#1374](https://github.com/rustic-rs/rustic/pull/1374))
- *(commands)* Use spawn_blocking in webdav when calling rustic_core ([#1365](https://github.com/rustic-rs/rustic/pull/1365))
- *(commands)* Prevent overwriting hot repository ([rustic_core](https://github.com/rustic-rs/rustic_core/releases/tag/rustic_core-v0.6.0)) in init command

### Miscellaneous

- update snapshots to include minutely configuration options
- *(deps)* update rustic_core, bytes, and libc dependencies to latest versions
- simplify lifetime annotations in OpenFileReader and TreeIterItem implementations
- clean up whitespace and update clippy linting allowances
- *(deps)* update dependencies to latest versions
- *(deps)* update lockfile to get rid of vulnerable `url` version
- *(commands)* rename fields for clarity in mount, add user options for mount ([#1353](https://github.com/rustic-rs/rustic/pull/1353))
- *(deps)* update dependencies
- *(deps)* don't use rustic_core webdav feature ([#1367](https://github.com/rustic-rs/rustic/pull/1367))
- move `webdavfs` from `rustic_core` to `rustic-rs` ([#1363](https://github.com/rustic-rs/rustic/pull/1363))
- *(clippy)* comment out unused lints in lib.rs
- *(clippy)* apply fixes automatically
- use BTreeMap for env in global options ([#1360](https://github.com/rustic-rs/rustic/pull/1360))
- add tiny framework for testing rustic's compat with latest restic ([#1303](https://github.com/rustic-rs/rustic/pull/1303))
- use snapshot tests for default config, show-config and completions ([#1359](https://github.com/rustic-rs/rustic/pull/1359))
- *(deps)* update dependencies rustic_core, rustic_backend, rustic_testing, and migrate to conflate 0.3 ([#1357](https://github.com/rustic-rs/rustic/pull/1357))
- fix typos
- *(build)* add platform-dependent settings and remove ci flag for extra features
- clarify `--use-profile` command in config by using long form ([#1344](https://github.com/rustic-rs/rustic/pull/1344))
- *(deps)* update core and testing crates ([#1340](https://github.com/rustic-rs/rustic/pull/1340))

<hr /><em>This discussion was created from the release <a href='https://github.com/rustic-rs/rustic/releases/tag/v0.9.5'>v0.9.5</a>.</em>

**Comment by lebuzzer** [comment]
hello and happy new year to team :)
thank you all for your work.
I do not see armv7 build for this release. is it normal ?

**Reply by simonsan** [reply]
Yes, there were problems with some of our dependencies not building on it, afair. So it has been temporarily suspended: https://github.com/rustic-rs/rustic/blob/main/.github/workflows/release-cd.yml#L95
